src/graphql.ts

```diff
@@ -127,7 +127,10 @@ const createGraphQLHandler = (operationType: OperationTypeNode, mask: Mask) => {
           }
 
           case 'POST': {
-            if (!req.body) {
+            if (
+              !req.body ||
+              !(req.body as GraphQLRequestPayload<VariablesType>).query
```

Comment:
Let's use optional chaining here instead to keep a shorter notation:

```js
if (!req.body?.query) {
```

Once you do this, you'll get an error that `query` doesn't exist on type `string`. That is because the `parse` function uses a `req: MockedRequest` type, but instead it should respect the custom request type provided by the request handler. To fix this, go to `src/handlers/requestHandler.ts` and change:

```diff
-parse?: (req: MockedRequest) => ParsedRequest | null
+parse?: (req: RequestType) => ParsedRequest | null
```

Let me know if you have any questions.